userspace/libsinsp/container_engine/docker_common.cpp

```diff
@@ -319,18 +320,21 @@ bool docker_async_source::parse_docker(std::string &container_id, sinsp_containe
 			Json::Value img_root;
 			if(reader.parse(img_json, img_root))
 			{
+			    unordered_set<std::string> imageDigestSet;
 				for(const auto& rdig : img_root["RepoDigests"])
```

Comment:
Can you please add a comment here explaining that `img_root["RepoDigests"]` only contains entries only for images pulled from a registry, so:
 - due to re-tagging we may not find `container->m_imagerepo` in the list
 - for locally built images the list will be empty

?

Reply:
yup done.